hugegraph-dist/src/main/java/com/baidu/hugegraph/dist/HugeGraphServer.java

```diff
@@ -41,25 +41,34 @@
     }
 
     public static void main(String[] args) throws Exception {
-
         if (args.length != 2) {
             String msg = "HugeGraphServer can only accept two config files";
             LOG.error(msg);
             throw new HugeException(msg);
         }
 
+        RegisterUtil.registerBackends();
+        // Scan the jars in plugins directory and load them
+        registerPlugins();
+
```

Comment:
add `HugeRestServer.register()` and `HugeGremlinServer.register()` to do:
```
RegisterUtil.registerBackends();
RegisterUtil.registerPlugins();
```

just call `HugeRestServer.register()` here